tests/support/StubComposerRunner.php

```diff
@@ -19,7 +19,7 @@ public function __construct(
     public function licenses(string $projectPath): string
     {
         if ($this->willThrow) {
-            throw FailedRunningComposer::withReason($this->output === '' ? 'unknown' : $this->output);
+            throw FailedRunningComposer::withCommand($this->output === '' ? 'unknown' : $this->output);
```

Comment:
This is wrong but it's fine I think, we can always fix up later as it's just the stub impl.